components/prism-php.js

```diff
@@ -139,10 +158,42 @@
 			lookbehind: true
 		},
 		'number': /\b0b[01]+\b|\b0x[\da-f]+\b|(?:\b\d+(?:_\d+)*\.?(?:\d+(?:_\d+)*)*|\B\.\d+)(?:e[+-]?\d+)?/i,
-		'operator': /<?=>|\?\?=?|\.{3}|->|[!=]=?=?|::|\*\*=?|--|\+\+|&&|\|\||[?~]|[/^|%*&<>.+-]=?/,
+		'operator': /<?=>|\?\?=?|\.{3}|\??->|[!=]=?=?|::|\*\*=?|--|\+\+|&&|\|\||<<|>>|[?~]|[/^|%*&<>.+-]=?/,
 		'punctuation': /[{}\[\](),:;]/
 	};
 
+	Prism.languages.insertBefore('php', 'comment', {
+		'attribute': {
+			pattern: /#\[(?:(?!#\[)(?:\\[\s\S]|[^\\\]]))*]/,
+			greedy: true,
+			inside: {
+				'interpolation': {
+					pattern: /\[(?:(?!#\[)(?:\\[\s\S]|[^\\\]]))*]/,
+					inside: {
+						rest: Prism.languages.php,
```

Comment:
Please move `rest` below `attribute-class-name`. `rest` will cause the token of `Prism.languages.php` to be inserted at the end of the `inside` object. Having it here is misleading.

Reply:
fixed